make_doc.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/sh
```

Comment:
```suggestion
#!/usr/bin/env sh
```

In my environment the previous  shearline was producing errors, but this one works.